packages/vscode-extension/src/debug/depsChecker/spfxNodeChecker.ts

```diff
@@ -2,8 +2,14 @@ import { DepsCheckerEvent, nodeNotFoundHelpLink, nodeNotSupportedForSPFxHelpLink
 import { NodeChecker } from "./nodeChecker";
 
 export class SPFxNodeChecker extends NodeChecker {
-  protected readonly _supportedVersions: string[] = ["10", "12", "14"];
   protected readonly _nodeNotFoundHelpLink = nodeNotFoundHelpLink;
-  protected readonly _nodeNotSupportedHelpLink = nodeNotSupportedForSPFxHelpLink;
   protected readonly _nodeNotSupportedEvent = DepsCheckerEvent.nodeNotSupportedForSPFx;
+
+  protected async getNodeNotSupportedHelpLink(): Promise<string> {
```

Comment:
Does it need to be `async`?

Reply:
1. In AzureNodeChecker's implementation, we need to read config files to determine whether it contains function. So it has to return a Promise. So the base class needs to return Promise
2. In SpfxNodeChecker, it does not have to async. We can also `return Promise.resolve()` but I think these two are not much of a difference